src/utils/fileUtils.js

```diff
@@ -56,6 +56,18 @@ const extractFilePaths = function(path) {
 	return [dirPath, fileName]
 }
 
+/**
+ * A single shared file can be opend with path: '/'.
+ * In this case there is no parent folder to find sibling files in.
+ *
+ * @param {string} path the full path
+ * @return {boolean}
+ */
+const isSingleSharedFile = function(path) {
+
+	return path === '/'
+}
```

Comment:
Following other comments, this would therefor be obsolete https://github.com/nextcloud/viewer/pull/1242/files#r876750069

Reply:
✅